### ISSUE TITLE
Disabling options not working

### ISSUE THREAD

**Issue by rudachenkoev** [ranbuch/accessibility#61]
#56 
Disabling the `increaseLineHeight` and `decreaseLineHeight` options does not work. I suspect this is because the `IAccessibilityModulesOptions` interface lacks these parameters.

```
new Accessibility({
  modules: {
    increaseLineHeight: false,
    decreaseLineHeight: false
  }
})
```

**Comment by ranbuch** [comment]
@rudachenkoev you are correct. Please try version `4.5.12`, I'll push to repo soon but `npm` is already updated.
